R/ff_prep_refactored.R

```diff
@@ -62,7 +62,10 @@
 #'
 #' @keywords machine-learning data-preparation forestry
 
-ff_prep_refactored <- function(datafolder = NA, country = NA, shape = NA, tiles = NULL, groundtruth_pattern = "groundtruth6m", dates = "2023-01-01",
+# source(system.file("R", "load_config.R", package = "ForestForesight"))
```

Comment:
what is this comment?

Reply:
I should delete this. This is from my experiment to make config file run in testthat